### ISSUE TITLE
Question regarding inputting range in stat block

### ISSUE THREAD

**Issue by rjbprime** [mlenser/5eMonsterConverterCompiled#1]
Hi mate,

just a quick query. How does one input a range on a ranged attack? Tried using the Hill Giant stat block, which has a ranged rock throwing attack, but the section for this line doesn't have a range, its in the special qualities section. Tried copying the distance to the ranged attack line, in different places, can't get it to show in the stat block on the right.

Pleas advise.

Kind regards,

RJBPrime

**Comment by mlenser** [comment]
My script can't handle special ranges set up that way currently.

The script, in its current form, won't be updated. I may fully redo it over a weekend or something so I can host it on my new site and can fix issues with it.

**Comment by rjbprime** [comment]
Fair enough. For now I'll just use it to convert PF1e monsters to 5e, then use the converted creature in a 5e specific stat block. If the critter has ranged attacks, I'll import it's range into the stat block I'm using.

Thanks for the heads up on redoing the script, would you be able to notify me when it's done? As its a very useful script, besides the range issue.

**Comment by mlenser** [comment]
I'll make a post here if I do it.

**Comment by rjbprime** [comment]
Cheers for that.

**Comment by mlenser** [comment]
I've ported my old code to my new codebase: https://marklenser.com/5e/converters/pathfinder-to-5e

I've made some improvements (while likely breaking some thigns), but range is still not supported.
